PurchasesCoreSwift/Attribution/ASIdentifierManagerProxy.swift

```diff
@@ -29,9 +29,7 @@ class FakeASIdentifierManager: NSObject {
 
 }
 
-// TODO (post-migration): Make all the things internal again.
-@objc(RCASIdentifierManagerProxy)
-public class ASIdentifierManagerProxy: NSObject {
+class ASIdentifierManagerProxy {
```

Comment:
Is this change safe? I think so because we don't use `unsafeBitCast` to call this class

Reply:
🤔 I'm not sure that's the case...
We're using `NSClassFromString`, and relying on this [behavior](https://developer.apple.com/documentation/swift/anyclass) from the AnyClass type, where all the `@objc` stuff is available as optionals. 
So I _think_ we actually still need this to be `@objc`, but the only way to be certain is to test it out on device

Reply:
ok, I am going to revert the changes in these files since it looks like they require some testing I haven't done

Reply:
Good point @aboedo. Actually, as you said we are taking advantage of this [behavior](https://developer.apple.com/documentation/swift/anyclass) but the proxies don't expose the methods and properties we need, we have "**Fake**" clients that expose these methods instead. So those classes (`FakeAdClient`,` FakeTrackingManager` and `FakeASIdentifierManager`) must use`NSObject` and `@objc`.

But anyway, I agree with both of you, this is very tricky code and a few more tests wouldn't be bad at all 🙂

PS: Sorry for the inconvenience 😅

Reply:
@juanpe nothing to apologize for! You've helped us a ton, especially with the migration of this class! 
@vegaro could you add an issue to test this out and clean up the `@objc` if it's not necessary here?